pkg/declcfg/addTocfg.go

```diff
@@ -0,0 +1,104 @@
+package declcfg
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+
+	"github.com/operator-framework/operator-registry/pkg/image"
+	"github.com/operator-framework/operator-registry/pkg/registry"
```

Comment:
>I see the functionality present in this file as more of a client of pkg/declcfg

That is a really good point 

>Here are some alternate suggestions:
./cmd/opm/add/request.go
./pkg/action/add_bundles_to_configs.go
With option 1, we're basically saying this functionality is an implementation detail of the CLI, so it belongs alongside the CLI code in the cmd/opm/add package.
With option 2, we're saying a library of "actions" that the CLI uses, but other clients could use those actions too. This is how the Helm CLI is setup, and is a pattern I followed in the kubectl operator code.

Both the options sounds really good, so it's hard to choose one over the other just based on the meaning of choosing each option that's laid out here. In lieu of that, going with option 2 in the interest of standardizing design across projects unless I hear any objections.

Reply:
Thinking more, if we want to align actions to CLI commands, perhaps just `./pkg/action/add.go`?

Reply:
I've actually seen `pkg/cmd` used in a lot of kubectl plugins/utilities build with cobra. It seems relatively clean to me.

e.g. https://github.com/njhale/kubectl-stash/blob/master/pkg/cmd/get.go

Reply:
That works too! I don't have a strong opinion on what package it ends up in. For me the main points are:
1. This package is completely standalone from the CLI implementation (i.e. it shouldn't mention cobra or flags, IMO, that's what `cmd/opm/add.go` is for).
2. It's separate for the underlying building blocks it uses (e.g. `pkg/declcfg`, `pkg/registry`, etc.)
3. It's designed such that it can be used as a standalone library. Come at it from the mindset of someone who needs this functionality vendored into their other non-operator-registry tool.